src/redux/user/user.types.js

```diff
@@ -2,6 +2,7 @@ export const SET_USER = 'SET_USER';
 export const SET_USER_ERROR = 'SET_USER_ERROR';
 export const SET_USER_LOADING = 'SET_USER_LOADING';
 export const LOGIN_USER = 'LOGIN_USER';
+export const GOOGLE_USER = 'GOOGLE_USER';
```

Comment:
You need to use a more understandable type name for this action (like LOGIN_USER_BY_GOOGLE or LOGIN_BY_GOOGLE)

Reply:
Fixed